kornia/contrib/models/sam/model.py

```diff
@@ -31,6 +32,7 @@ class SamModelType(Enum):
     vit_h = 0
     vit_l = 1
     vit_b = 2
+    mobile_sam = 3
```

Comment:
There is ViT-Ti (which I assume stands for Tiny), first introduced by DeiT (https://arxiv.org/pdf/2012.12877.pdf). Some subsequent ViT papers from Google also acknowledge this variant (https://arxiv.org/pdf/2106.10270.pdf, https://arxiv.org/pdf/2106.04560.pdf). Thus, I'm a bit hesitant to use the name `vit_tiny` as it can be confused with ViT-Ti. Perhaps we can change it to `tiny_vit` instead?